### PR TITLE
Switch to specific components

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: components/components.yaml [hunk 1]
```diff
@@ -724,3 +724,63 @@
   ProjectUrl: https://github.com/Particular/NServiceBus.TransactionalSession
   NugetOrder:
     - NServiceBus.TransactionalSession
+
+- Key: TransactionalSession.NHibernate
+  Name: NServiceBus.NHibernate.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/nhibernate
+  GitHubOwner: Particular
+  Category: Other
```
Comment (danielmarbach) [inline components/components.yaml:732]: Should we use

```suggestion
  Category: Persistence
```

?

File: components/components.yaml [hunk 2]
```diff
@@ -724,3 +724,63 @@
...
+  LicenseUrl: https://github.com/Particular/NServiceBus.Persistence.Sql/blob/master/LICENSE.md
+  ProjectUrl: https://github.com/Particular/NServiceBus.Persistence.Sql
+  NugetOrder:
+    - NServiceBus.Persistence.Sql.TransactionalSession
+
+- Key: TransactionalSession.AzureTable
+  Name: NServiceBus.Persistence.AzureTable.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/azure-table
+  GitHubOwner: Particular
+  Category: Other
```
Comment (danielmarbach) [inline components/components.yaml:752]:
```suggestion
  Category: Persistence
```

File: components/components.yaml [hunk 3]
```diff
@@ -724,3 +724,63 @@
...
+  LicenseUrl: https://github.com/Particular/NServiceBus.Persistence.AzureTable/blob/master/LICENSE.md
+  ProjectUrl: https://github.com/Particular/NServiceBus.Persistence.AzureTable
+  NugetOrder:
+    - NServiceBus.Persistence.AzureTable.TransactionalSession
+
+- Key: TransactionalSession.CosmosDB
+  Name: NServiceBus.Persistence.CosmosDB.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/cosmosdb
+  GitHubOwner: Particular
+  Category: Other
```
Comment (danielmarbach) [inline components/components.yaml:762]:
```suggestion
  Category: Persistence
```

File: components/components.yaml [hunk 4]
```diff
@@ -724,3 +724,63 @@
...
+  LicenseUrl: https://github.com/Particular/NServiceBus.Persistence.CosmosDB/blob/master/LICENSE.md
+  ProjectUrl: https://github.com/Particular/NServiceBus.Persistence.CosmosDB
+  NugetOrder:
+    - NServiceBus.Persistence.CosmosDB.TransactionalSession
+
+- Key: TransactionalSession.MongoDB
+  Name: NServiceBus.Storage.MongoDB.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/mongodb
+  GitHubOwner: Particular
+  Category: Other
```
Comment (danielmarbach) [inline components/components.yaml:772]:
```suggestion
  Category: Persistence
```

File: components/components.yaml [hunk 5]
```diff
@@ -724,3 +724,63 @@
...
+  LicenseUrl: https://github.com/Particular/NServiceBus.Storage.MongoDB/blob/master/LICENSE.md
+  ProjectUrl: https://github.com/Particular/NServiceBus.Storage.MongoDB
+  NugetOrder:
+    - NServiceBus.Storage.MongoDB.TransactionalSession
+
+- Key: TransactionalSession.RavenDB
+  Name: NServiceBus.RavenDB.TransactionalSession
+  DocsUrl: /nservicebus/transactional-session/persistences/ravendb
+  GitHubOwner: Particular
+  Category: Other
```
Comment (danielmarbach) [inline components/components.yaml:782]:
```suggestion
  Category: Persistence
```

File: components/nugetAlias.txt [hunk 6]
```diff
@@ -88,4 +88,10 @@ Extensions.Logging: NServiceBus.Extensions.Logging
 TimeoutMigration: Particular.TimeoutMigration
 CosmosDB: NServiceBus.Persistence.CosmosDB
 NonDurablePersistence: NServiceBus.Persistence.NonDurable
-TransactionalSession: NServiceBus.TransactionalSession
\ No newline at end of file
+TransactionalSession: NServiceBus.TransactionalSession
```
Comment (danielmarbach) [inline components/nugetAlias.txt:91]: This is just a thought. Do we consider grouping the tx session related stuff more important or the fact that persistence and tx session go hand in hand? For the later, we could add the persistence-specific component alias to the persistence. Same for the component definitions

Comment (SzymonPobiega) [reply components/nugetAlias.txt:91]: I have no strong opinion. Happy to move it around if rest of you folks agree.

Comment (danielmarbach) [reply components/nugetAlias.txt:91]: Me neither. I'm good either way. I just wanted to share my observations and thoughts

**Review by tmasternak — APPROVED** [review]

**Review by danielmarbach — APPROVED** [review]

**Pull request merged**